src/simplifiers/LA.cc

```diff
@@ -151,43 +143,40 @@ void LAExpression::print(ostream & os) const {
     else {
         // There is at least one variable
         os << " (+";
-        for (auto const & term: polynome) {
-            if (term.first == PTRef_Undef)
-                constant = -term.second;
+        for (auto const & [var, factor] : polynome) {
+            if (var == PTRef_Undef)
+                constant = - factor;
             else
-                os << " (* " << term.second << " " << logic.printTerm(term.first) << ")";
+                os << " (* " << factor << " " << logic.printTerm(var) << ")";
         }
         os << ")";
     }
     if (r == OP::EQ || r == OP::LEQ)
         os << " " << constant << ")";
-    os << '\n';
+    os << std::endl;
```

Comment:
This is the only thing I would probably not do.
If you are passed a stream, it does not feel like the responsibility of this function to flush the stream.
Maybe the caller does not want that. I would keep `\n` here.

Reply:
Ok, I'll take it off!